modules/nw-networkpolicy-audit-concept.adoc

```diff
@@ -36,18 +45,39 @@ The logging format is compatible with syslog as defined by RFC5424. The syslog f
 2023-11-02T16:28:57.235Z|00006|acl_log(ovn_pinctrl0)|INFO|name="NP:verify-audit-logging:Ingress", verdict=drop, severity=alert, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:81:02:01,dl_dst=0a:58:0a:81:02:23,nw_src=10.131.0.39,nw_dst=10.129.2.35,nw_tos=0,nw_ecn=0,nw_ttl=62,nw_frag=no,tp_src=58496,tp_dst=8080,tcp_flags=syn
 ----
 
+You can configure log file parameters for the OVN-Kubernetes network plugin. The following example lists these parameters and their default values:
+
+[source,go]
```

Comment:
[AsciiDoc](https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-highlighter/#apply-source-highlighting) content points to Rogue documentation and `go` is listed there.

Reply:
This is what is manipulated on cluster to change configuration,  forward logs to syslog or journald
<pre>
 policyAuditConfig:
          destination: "null"
          maxFileSize: 50
          maxLogFiles: 5
          rateLimit: 20
          syslogFacility: local0
</pre>

Reply:
:heavy_check_mark: